docs/API.md

```diff
@@ -895,19 +903,16 @@ The value of `"test_params"` is an object with the following properties:
 * `"config"`: A string, optional. The name of a *config profile*.
 * `"user_ip"`: A ..., optional.
 * `"user_location_info"`: A ..., optional.
-* `"priority"`: An integer, optional
-* `"queue"`: An integer, optional
+* `"priority"`: A *priorty*, optional
+* `"queue"`: An *queue*, optional
 
-*priotity*
+priority parameter
```

Comment:
This is section is redundant since the data type is already pointed out in the params list above.